config/dev/dgutman-dsa-server-preprod-ec2.yaml

```diff
@@ -0,0 +1,27 @@
+# This stack is based on `config/prod/dgutman-htan-new2-ec2.yaml`
+# in https://github.com/Sage-Bionetworks/sandbox-provisioner/
+template_path: "remote/managed-ec2-linux-v2.j2"
+stack_name: "dgutman-dsa-server-preprod-ec2"
+
+sceptre_user_data:
+  Distro: "ubuntu"
+  OpenPorts: ["80", "443", "22"]
```

Comment:
Port 80 should not be only on this instance. If this is a web app that will be used by anyone except developers, it should have an ELB in front and the instance should not have to do any 80->443 redirect. If it truly is just for dev, this is probably ok to open these ports, but it should not be considered secure. IOW, this is not a configuration we would replicate in prod.

Reply:
I've noted this requirement for the production template [here](https://sagebionetworks.jira.com/browse/IT-1214?focusedCommentId=130370).